CHANGELOG.md

```diff
@@ -6,6 +6,26 @@ For advice on how to use these release notes see [our guidance on staying up to
 
 ### New features
 
+#### We've adjusted our responsive type scale
+
+We've made the following adjustments to our responsive type scale:
+
+- point 16 is now 16px across all screen sizes
```

Comment:
I've had a rethink about these. Have a look at the following suggestions and see if it works.
```suggestion
- 16px is now the same across all screen sizes
```

Reply:
This is a tricky one because that first subject, "point 16", isn't representative of the specific unit 16px necessarily. It's point "16" on our responsive type scale. [Chris's rewrite of the typography section best illustrates this](https://deploy-preview-3343--govuk-design-system-preview.netlify.app/styles/type-scale/). For extra context, 16 used to be 16px on big screens, 14px on small screens but this new change means it's 16px wholesale. I wanted to try and separate the point on the scale and the font size it returns which is really bothersome considering they're numbers that could be mistaked for the specific font size they relate to 🤯

Maybe something like "point 16 **now returns** 16px across all screen sizes"? It makes it clearer that point 16 isn't the same as 16px but that it is linked to it. I'm not super confident about that, hopefully it'll spark inspiration for your big content brain.

Reply:
Your suggestion looks like it'll work well. I shall edit...